src/Token.react.js

```diff
@@ -23,22 +23,29 @@ class Token extends React.Component {
   _renderRemoveableToken = () => {
     const {active, children, className, onRemove, ...props} = this.props;
 
+    /* eslint-disable jsx-a11y/interactive-supports-focus */
     return (
       <div
         {...props}
         className={cx('rbt-token', 'rbt-token-removeable', {
           'rbt-token-active': active,
-        }, className)}>
+        }, className)}
+        onClick={onRemove}
+        onKeyDown={this._handleRemoveButtonKeydown}
+        role="button">
+        <span className="sr-only">
```

Comment:
What's happening here? It looks like you've added a separate, a11y-only version of the clear button in addition to the existing one. `ClearButton` should have pretty good a11y, but if there's something off, it would be better to modify that instead.